frontend/src/metabase/entities/tables.js

```diff
@@ -167,6 +169,20 @@ const Tables = createEntity({
       }
 
       if (state[virtualQuestionId]) {
```

Comment:
I honestly think there is an arguement to be made to just always return `convertSavedQuestionToVirtualTable(card)`, but this will keep the simpler properties synced up

Reply:
Can't really remember the reason I added this check originally. It was added during the original models implementation in #18993, so it can be related to a flow when you turn a question into a model or vice-versa. However, I tried things out locally, and it seems to work fine. My second guess is that the "table" record we fetch from the server might contain more accurate data than what `convertSavedQuestionToVirtualTable` can generate, but I'm not sure I can see what exactly is missing. 

Really appreciate a safeguard there with properties check and `updateIn` approach. I'd leave it like that to play it safe